sw/device/silicon_creator/rom/keys/README.md

```diff
@@ -1,8 +1,53 @@
 # Signing Keys
 
-This directory contains ASN1 DER encoded development keys for signing ROM\_EXT
-images. Until we have a more ergonomic tool for working with keys and signatures,
-below snippets can be used for various operations.
+# Generating keys with opentitantool
+
+Open Titan Tool is the recommended tool to generate RSA keys to use in
+the `ROM`.
+
+- Inspecting a key (show the public information of a key):
+```
+$ bazel run //sw/host/opentitantool -- rsa key show $PWD/<basename>.der
+{
+  key_num_bits: 3072,
+  modulus: "0xa22244f7...",
+  public_exponent: "0x010001",
+  n0_inv: "0x2af3...",
+  rr: "0x82ca57ba..."
+}
+```
+
+- Generating a key pair in the current directory:
+```
+$ bazel run //sw/host/opentitantool -- rsa key generate <outdir> <basename>
+```
+This will generate two files `<outdir>/<basename>.der` (private key)
+and `<outdir>/<basename>.pub.der` (public key). Note that if you want to generate
+a key in the current directory, you need to set `<outdir>` to `$PWD` since `bazel` does
+not run the tool in the current directory.
+
+- Exporting a key to a header for use in the ROM:
+```
+$ bazel run //sw/host/opentitantool -- rsa key export $PWD/<basename>.der
+exporting key to .../<basename>.h
+$
+```
+After exporting the key, it is recommended to follow the best practices and make
+sure that the header follows the OpenTitan convention (for example for fake
+keys that will be committed to the repository) by running
+```
+# from root of the repository
+$ ./util/fix_include_guard.py sw/device/.../<basename>.h
+$ bazel run //quality:clang_format_fix
+```
+
+**Remark**: it is recommended to specify the licence at the beginning of the file
```

Comment:
nit: Since presubmit fail you can just say that, i.e. don't forget to add the copyright comment at the beginning of the file.